packages/p2p-v2/src/components/PaymentMethodForm/PaymentMethodForm.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { TFormState } from 'src/reducers/types';
```

Comment:
didn't this import give any issues?

Reply:
Yeap @nada-deriv, If this `import { TFormState } from 'src/reducers/types';`  is the import you mean, then yes, it works just fine